exercise_10/transformCollectionToString.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Write JS function "transformCollectionToString" to transform collection into string
+    - if array has elements, then return array as string, joined by '|' (pipe character)
+    - otherwise, return an empty string
+ * input: array (collection) 
+ * output: string
+ */
+
+/**
+ * CASE #1
+ * input: ['apple', 'grapes', 'strawberries']
+ * output: "apple | grapes | strawberries"
+ */
+
+/**
+ * CASE #2
+ * input: []
+ * output: ""
+ */
+
+/**
+ *
+ * @param {Array} collection
+ * @returns {string}
+ */
+function transformCollectionToString(collection) {
+  if (Array.isArray(collection) && collection.length > 0) {
```

Comment:
We defined a method to check wether a collection has values or not.

FIX: use abstractions, not implementations.